birdhouse/config/magpie/providers.cfg.template

```diff
@@ -62,6 +62,31 @@ providers:
     c4i: false
     type: thredds
     sync_type: thredds
+    # below is a custom config to indicate how magpie should convert thredds path elements into resources/permissions
+    # see: https://pavics-magpie.readthedocs.io/en/latest/services.html#servicethredds
+    configuration:
+      skip_prefix: "thredds"  # prefix to ignore, below prefixes will be matched against whatever comes after in path
+      file_patterns:
+        # note: make sure to employ quotes and double escapes to avoid parsing YAML error
+        - ".+\\.nc"
```

Comment:
Looks good now but you sure `.ncml` do not need to be added to this list, as I mentioned here https://github.com/bird-house/birdhouse-deploy/pull/107#discussion_r530617162?  Your call, you know this best.

Reply:
Depends what we want to achieve. 
If we want to create 2 "resources", one of `.nc` and another for `.ncml`, it should be added. 
But if we want to have the same "permission set" for both these extension as if they were the same file, then this regex does just that.

Reply:
done 
f448778